File: src/main/java/ru/dragonestia/apocalypse/item/SulfurClusterItem.java
package ru.dragonestia.apocalypse.item;

import cn.nukkit.item.Item;
import ru.dragonestia.expo.item.AdvItem;

public class SulfurClusterItem extends Item implements AdvItem {

    public SulfurClusterItem(Integer meta, int count) {
        super(ApocalypseID.SULFUR_CLUSTER, meta, count, "Sulfur cluster");
    }

    @Override
    public String getRuName() {
        return "Серная руда";
    }

    @Override
    public String getIcon() {
        return "textures/items/cluster/sulfur";
    }

}

File: src/main/java/ru/dragonestia/apocalypse/item/SulfurClusterItem.kt
package ru.dragonestia.apocalypse.item

import cn.nukkit.item.Item
import ru.dragonestia.expo.item.AdvItem

class SulfurClusterItem @JvmOverloads constructor(meta: Int? = 0, count: Int = 1) : Item(ApocalypseID.SULFUR_CLUSTER, 0, count, "Sulfur cluster"), AdvItem {

    override fun getRuName(): String = "Серная руда"

    override fun getIcon(): String = "textures/items/cluster/sulfur"

}